files/zh-cn/mdn/writing_guidelines/howto/markdown_in_mdn/index.md

```diff
@@ -515,7 +526,7 @@ GFM 表格语法的主要限制是：
 因为它们有一个标题列，GFM 无法表示这些页面。因此，应该使用 HTML。为了获得特殊的样式，还需要将 `"properties"` 类应用于表格：
 
 ```html
-<table class="properties">
+<table class="properties"></table>
```

Comment:
also `nolint` here

Reply:
I noticed that upstream also had a `-nolint`, but it doesn't feel like the proper use of `-nolint`, so I'm removing it from upstream and formatting the code block like this in the English content!